src/Core/ConstraintValidator/Constraints/IsUrlRewrite.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints;
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\IsUrlRewriteValidator;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Class IsUrlRewrite.
+ *
+ * @Annotation
+ */
+class IsUrlRewrite extends Constraint
+{
+    // todo: test maybe similar translations already exist
+    public $message = 'Value "%value%" is invalid.';
```

Comment:
Hi @LouiseBonnard  - I'm thinking if such translation already exist somewhere. If not, then I assigned it to the domain 'Admin.Notifications.Error'.

for my use case this translation is used to display an error message here

![link_rewrite_errors](https://user-images.githubusercontent.com/17051250/50000120-3f342680-ffa1-11e8-828b-8c084d629577.png)


is it good or I should consider to display different message?

Reply:
surely it does - thank you! :+1:

Reply:
@matks, can you confirm that `%s` here can stand for `%value%`?

Reply:
@LouiseBonnard Yes, we can use `%s` or `%value` bot are valid

In the code, we "link" the translated message later with what has to be replaced, like this:
```
buildViolation($message)
                ->setTranslationDomain('Admin.Notifications.Error')
                ->setParameter('%s', $value)
```

Reply:
Great, thanks @matks!